### ISSUE TITLE
Interactive Controls must not be nested accessibility issue

### ISSUE THREAD

**Issue by pdfabbro** [KarimMokhtar/react-drag-drop-files#157]
When you run axe devTools on the file upload component it complains with "Interactive controls must not be nested" due to the label element having role="button" on it.  This combined with the hidden input field lead to the accessibility issue.  role="button" should be removed from the label.  Either that or we should be able to override this property if we provide a custom button as children

**Comment by sendmenas** [comment]
Issue still persist 🫤
